docs-new/docs/contribute/docs/spell-check.md

```diff
@@ -0,0 +1,76 @@
+# Spell checker
+
+All PRs that are pushed to a Keptn repository
+are run through a spell checker that is based on the
+[check-spelling](https://github.com/check-spelling/check-spelling)
+GitHub action.
+
+Note, however, that you are still responsible for reading your text carefully.
+The tool does not flag if you used a real word that is not the right word,
+if you misuse "setup" versus "set up", and so forth.
+
+## Handling Spell Checker errors
+
+If you get a Spell Checker error:
+
+1. Right-click the `details` link for Spell Checker.
+1. If you do not want to read through the whole (and rather verbose) log,
+   type `unrecognized words` in the search box at the top.
```

Comment:
i suggest to point users to the job summary that is nicely rendered and divided into general and details errors

Reply:
How do they get to the job summary?

Reply:
Done.